gui/ThemeEngine.cpp

```diff
@@ -234,10 +234,8 @@ ThemeEngine::ThemeEngine(Common::String id, GraphicsMode mode) :
 	_cursorHotspotX = _cursorHotspotY = 0;
 	_cursorWidth = _cursorHeight = 0;
 	_cursorTransparent = 255;
-#ifndef USE_RGB_COLOR
```

Comment:
I don't quite understand this. Now, we force all cursors to CLUT8?

Reply:
These are just initial values. The actual format is set later when the image data is loaded.